src/ticketing/api.ts

```diff
@@ -57,12 +58,19 @@ export async function listRecurringPayments(): Promise<RecurringPayment[]> {
 
 export async function addPaymentMethod(paymentRedirectUrl: string) {
   const url = `ticket/v3/recurring-payments`;
-  const response = await client.post<AddPaymentMethodResponse>(
+  return client.post<AddPaymentMethodResponse>(
     url,
     {paymentRedirectUrl},
     {authWithIdToken: true},
-  );
-  return response.data;
+  ).then(async (response) => {
+    await InAppBrowser.open(
+      response.data.terminal_url,
+      // Param showInRecents is needed so the InAppBrowser doesn't get closed when the app goes to background
+      // hence user is again navigated back to browser after finishing the Nets flow,
+      // and then can complete the authentication process successfully
+      {showInRecents: true},
+    );
+  });
```

Comment:
I think we don't want to trigger opening of the in-app-browser in the api module.

Reply:
I see, I only followed the Vipps Login flow on `src/api/vipps-login/api.ts`, I'll update it

Reply:
refactored